pkg/syncer/fullsync.go

```diff
@@ -50,24 +50,34 @@ func csiFullSync(ctx context.Context, metadataSyncer *metadataSyncInformer) {
 
 	// k8sPVMap is useful for clean and quicker look up.
 	k8sPVMap := make(map[string]string)
+	var volumeMigrationService migration.VolumeMigrationService
```

Comment:
We have `volumeMigrationService` accessible within the syncer package. We do not need a local variable.

Reply:
Done, removed